puppet/zulip/templates/uwsgi.ini.template.erb

```diff
@@ -23,6 +23,10 @@ master-fifo=/home/zulip/deployments/uwsgi-control
 # How many serving processes to fork
 processes=<%= @uwsgi_processes %>
 
+# Give the processes short process names
+auto-procname=true
+procname-prefix-spaced=zulip-django
```

Comment:
This looks nice!  Maybe worth adding to the commit message what these looked like before.

Reply:
Also, does this adjust how you `pkill -f` these processes?

Reply:
Yes.  `pgrep -f uwsgi` will no longer match, but `pgrep -f zulip-django` will.  A grep doesn't find any place we use `pgrep` or `pkill` with django.

Reply:
Pushed a commit message change to show the old form.